tests/ui/blacklisted_name.rs

```diff
@@ -12,29 +12,32 @@ fn test(foo: ()) {}
 
 fn main() {
     let foo = 42;
-    let bar = 42;
     let baz = 42;
+    let quux = 42;
+    // Unlike these others, `bar` is considered an acceptable name to use.
+    // See https://github.com/rust-lang/rust-clippy/issues/5225.
```

Comment:
```suggestion
    // See https://github.com/rust-lang/rust-clippy/issues/5225.
    let bar = 42;
```

I would leave this in as a regression test

Reply:
Ah yes, that's a good plan, thank you.